tests/unit/admin/test-class-wc-rest-payments-accounts-controller.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Class WC_REST_Payments_Accounts_Controller_Test
+ *
+ * @package WooCommerce\Payments\Tests
+ */
+
+use PHPUnit\Framework\MockObject\MockObject;
+use WCPay\Exceptions\API_Exception;
+
+/**
+ * WC_REST_Payments_Accounts_Controller unit tests.
+ */
+class WC_REST_Payments_Accounts_Controller_Test extends WP_UnitTestCase {
+	/**
+	 * Controller under test.
+	 *
+	 * @var WC_REST_Payments_Accounts_Controller
+	 */
+	private $controller;
+
+	/**
+	 * @var WC_Payments_API_Client|MockObject
+	 */
+	private $mock_api_client;
+
+	/**
+	 * @var WC_Payments_API_Client
+	 */
+	private $original_api_client;
+
+	public function setUp() {
+		parent::setUp();
+
+		// Set the user so that we can pass the authentication.
+		wp_set_current_user( 1 );
+
+		$this->mock_api_client = $this->createMock( WC_Payments_API_Client::class );
+		$this->controller      = new WC_REST_Payments_Accounts_Controller( $this->mock_api_client );
+
+		// Inject the mocked client into service (happens in WCPay int, so reflection here).
+		$account_service     = WC_Payments::get_account_service();
+		$property_reflection = ( new ReflectionClass( $account_service ) )->getProperty( 'payments_api_client' );
+		$property_reflection->setAccessible( true );
+		$this->original_api_client = $property_reflection->getValue( $account_service );
+		$property_reflection->setValue( $account_service, $this->mock_api_client );
+	}
+
+	public function terDown() {
+		parent::terDown();
```

Comment:
Typo. Should be `tearDown()`.

Reply:
Oops, fixed in 3bdd5485c90d41a963cca19e639eca0b89713df2